WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -155,6 +156,8 @@ enum FeatureFlag: Int, CaseIterable, OverrideableFlag {
             return false
         case .readerUserBlocking:
             return true
+        case .personalizeHomeTab:
+            return false
```

Comment:
When is it going to be a good time to enable it?

Reply:
Thanks for the clarification. It'd be nice to have a way to override it during the build process, for example, using a local configuration file added to gitignore.